wdl/src/main/scala/wdl/expression/ValueEvaluator.scala

```diff
@@ -117,6 +117,15 @@ case class ValueEvaluator(override val lookup: String => WomValue, override val
             WomType.homogeneousTypeFromValues(pairs.map(_._2))
           ).coerceRawValue(pairs.toMap)
         }
+      case a: Ast if a.isObjectLiteral =>
+        val evaluatedMap = a.getAttribute("map").astListAsVector map { kv =>
+          val key = kv.asInstanceOf[Ast].getAttribute("key").sourceString
+          val value = evaluate(kv.asInstanceOf[Ast].getAttribute("value"))
+          key -> value
+        }
+        TryUtil.sequence(evaluatedMap map { tuple => tuple._2.map((tuple._1, _)) }) map { pairs =>
```

Comment:
What does this line do?

Reply:
Baaaaaaah, @jbuabbud just forget you ever saw `TryUtil` :)

Reply:
@jbuabbud - feel free to ask follow ups...

- `evaluatedMap` starts as a `Map[X, Try[Y]]`
- The `map ( tuple => ...)` turns each tuple of `(X, Try[Y])` in the `Map` into a Try[(X, Y)], which turns the `Map[X, Try[Y]]` into a `Seq[Try[(X, Y)]]`
- The `TryUtil.sequence` turns the `Seq[Try[(X, Y)]]` into a `Try[Seq[(X, Y)]]`. In other words, take a list of things that might have failed and turn it into a single list that might be a failure. It's not a great thing to use (cf @geoffjentry) because if the `Seq` has more than one failure in it, the result will only carry forwards the first one.